run-warc-tests.sh

```diff
@@ -17,7 +17,7 @@ ARCHIVES="${SCRIPT_DIR}/ARCHIVES"
 
 SERVO_DIR=$(readlink -f "${SERVO_DIR:-../servo}")
 SERVO_CMD=("${SERVO_DIR}/mach" "run" "-r" "-z"
-  "--userscripts" "${SCRIPT_DIR}/user-agent-js"
+  "--userscripts=" "${SCRIPT_DIR}/user-agent-js"
   "--certificate-path" "${SCRIPT_DIR}/proxy-certs/pywb-ca.pem"
   "--pref" "dom.testperf.enabled")
```

Comment:
I suspect we will hit the same problem with all of these arguments. Let's merge them all with = and join the strings.

Reply:
Do we need to add "=" here too, as per userscripts?